learning/tasks.py

```diff
@@ -0,0 +1,25 @@
+import os
```

Comment:
Nie wiem, czy chcemy to wstawiać do apki learning - apka accounts wydaje się bardziej do tego pasować